Tests/ForageSDKTests/ForagePINTextFieldTests.swift

```diff
@@ -9,45 +9,45 @@ import XCTest
 import VGSCollectSDK
 @testable import ForageSDK
 
-final class ForagePINTextFieldTests: XCTestCase {
```

Comment:
The functionality that this test was checking for is no longer valid and I will write new tests for the delegate functions in my next PR.